### PR TITLE
Adds Aftermath Support, Fixes Custom Flag Templates

### LINKED ISSUE #724: HQ flag texture is never set
Some players would like to play with custom flags, changing the flag texture however is not possible, as the HQ flag texture is never set. Change that so players can use custom flags.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## What type of PR is this.
1. [x] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added support for the Aftermath mod to Antistasi's template system.
Fixed custom flags not working for Templates.
Aftermath activates if Aftermath and Unsung are detected. Unsung vehicles are utilized due to a lack of themed vehicles, but the cold war tech fits well with the Aftermath universe equipment and feel.
Workshop collection utilized located here:
https://steamcommunity.com/workshop/filedetails/?id=2410758646

### Please specify which Issue this PR Resolves.
Fixes #724

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

Todo:

- [x] Setup defines for OPFOR/BLUFOR/THREEFOR/Civilians
- [x] Add custom flags/markers for OPFOR/BLUFOR/THREEFOR
- [x] Fix custom flags not working by default on game start flagpoles
- [x] Replace civilian pool when Aftermath is loaded
- [x] Setup vehicles for BLUFOR
- [ ] Setup vehicles for OPFOR
- [ ] Setup vehicles for THREEFOR
- [ ] Add support for faction weapon pools to draw from(maybe ACE loadout support?)
- [ ] Setup weapon pools for factions
- [ ] Redo THREEFOR template classes
- [ ] Balance Testing
- [ ] Redo squad layouts for OPFOR/BLUFOR/THREEFOR
- [ ] Test on Dedicated Server software, functions on LAN hosting so far

### CODE REVIEW

File: A3-Antistasi/Templates/Aftermath/Aftermath_Reb.sqf [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+SDKFlag = "Flag_Syndikat_F";
+SDKFlagTexture = "Templates\Aftermath\khans_flag_antistasi.paa";
+
+typePetros = "khan_great_01_body";//classname for the unit that should be used for petros
+
+////////////////////////////////////
+//             UNITS             ///
+////////////////////////////////////
+//First Entry is Guerilla, Second Entry is Para/Military || These are only to define the uniforms they should wear at low and high training respectively.
+staticCrewTeamPlayer = "khan_great_01_body";//classname
```
Comment (Lazejun) [inline A3-Antistasi/Templates/Aftermath/Aftermath_Reb.sqf:14]: ALL these Units have to be Unique, not littlery all the same ones

**Comment by Iamgoofball** [comment]
Fixed custom flag support:
steamwebhelper_WVGWNjoHON
ApplicationFrameHost_qmP7gXYjqe
steamwebhelper_if0VcB9qEn

Flags no longer have to swap sides once before getting the correct flag icon, they will now start with the correct flag icon.

File: A3-Antistasi/Templates/Aftermath/Aftermath_Reb.sqf (hunk 1 above)
Comment (Iamgoofball) [reply A3-Antistasi/Templates/Aftermath/Aftermath_Reb.sqf:14]: aye, need to add redoing the rebel template to the todo list

**Comment by Lazejun** [comment]
Well as the Goal of Next Release is new Template system, this is not realy useful.

**Comment by Iamgoofball** [comment]
@Lazejun This is using the new template system in 2.4.

**Comment by Lazejun** [comment]
No it isnt, these are the old Templates.

**Comment by Iamgoofball** [comment]
Could you link me the design doc for the new template system so I can take a look at it? I can begin migration to the new template system.

**Comment by Lazejun** [comment]
https://github.com/official-antistasi-community/A3-Antistasi/tree/loadout_templates/A3-Antistasi/Templates/NewTemplates/%23Examples

**Comment by Iamgoofball** [comment]
Great, I can get migrated over to this within a few hours.

**Comment by Iamgoofball** [comment]
Closed in favor of #1758

**Pull request closed without merging**